tests/int/__snapshots__/pactGenerating.test.ts.snap

```diff
@@ -662,6 +662,437 @@ Object {
+            "regex": "^[0-9a-fA-F]+$",
+          },
+          "$.body.object.ipv4Field": Object {
+            "match": "regex",
+            "regex": "^(\\\\d{1,3}\\\\.)+\\\\d{1,3}$",
+          },
+          "$.body.object.ipv6Field": Object {
+            "match": "regex",
+            "regex": "^(([0-9a-fA-F]{1,4}:){7,7}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,7}:|([0-9a-fA-F]{1,4}:){1,6}:[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,5}(:[0-9a-fA-F]{1,4}){1,2}|([0-9a-fA-F]{1,4}:){1,4}(:[0-9a-fA-F]{1,4}){1,3}|([0-9a-fA-F]{1,4}:){1,3}(:[0-9a-fA-F]{1,4}){1,4}|([0-9a-fA-F]{1,4}:){1,2}(:[0-9a-fA-F]{1,4}){1,5}|[0-9a-fA-F]{1,4}:((:[0-9a-fA-F]{1,4}){1,6})|:((:[0-9a-fA-F]{1,4}){1,7}|:)|fe80:(:[0-9a-fA-F]{0,4}){0,4}%[0-9a-zA-Z]{1,}|::(ffff(:0{1,4}){0,1}:){0,1}((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\\\\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])|([0-9a-fA-F]{1,4}:){1,4}:((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\\\\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9]))$",
+          },
+          "$.body.object.isoDateField": Object {
+            "match": "regex",
+            "regex": "^([\\\\+-]?\\\\d{4}(?!\\\\d{2}\\\\b))((-?)((0[1-9]|1[0-2])(\\\\3([12]\\\\d|0[1-9]|3[01]))?|W([0-4]\\\\d|5[0-2])(-?[1-7])?|(00[1-9]|0[1-9]\\\\d|[12]\\\\d{2}|3([0-5]\\\\d|6[1-6])))?)$",
+          },
+          "$.body.object.isoDatetimeField": Object {
+            "match": "regex",
+            "regex": "^\\\\d{4}-[01]\\\\d-[0-3]\\\\dT[0-2]\\\\d:[0-5]\\\\d:[0-5]\\\\d([+-][0-2]\\\\d:[0-5]\\\\d|Z)$",
+          },
+          "$.body.object.isoDatetimeWithMillisField": Object {
+            "match": "regex",
+            "regex": "^\\\\d{4}-[01]\\\\d-[0-3]\\\\dT[0-2]\\\\d:[0-5]\\\\d:[0-5]\\\\d\\\\.\\\\d+([+-][0-2]\\\\d(:?[0-5]\\\\d)?|Z)$",
+          },
+          "$.body.object.isoTimeField": Object {
+            "match": "regex",
+            "regex": "^(T\\\\d\\\\d:\\\\d\\\\d(:\\\\d\\\\d)?(\\\\.\\\\d+)?(([+-]\\\\d\\\\d:\\\\d\\\\d)|Z)?)?$",
+          },
+          "$.body.object.matcherWithExample1": Object {
+            "match": "regex",
+            "regex": "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+.[a-zA-Z]{2,}$",
+          },
+          "$.body.object.matcherWithExample2": Object {
+            "match": "regex",
+            "regex": "^[0-9a-fA-F]+$",
+          },
+          "$.body.object.timestampField": Object {
+            "match": "regex",
+            "regex": "^(Mon|Tue|Wed|Thu|Fri|Sat|Sun),\\\\s\\\\d{2}\\\\s(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)\\\\s\\\\d{4}\\\\s\\\\d{2}:\\\\d{2}:\\\\d{2}\\\\s(\\\\+|-)\\\\d{4}$",
+          },
+          "$.body.object.uuidField": Object {
+            "match": "regex",
+            "regex": "^[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}$",
+          },
+          "$.body.objectInline.matcher": Object {
+            "match": "regex",
+            "regex": "^[0-9a-fA-F]+$",
+          },
+          "$.body.objectInline.nestedArray": Object {
+            "match": "regex",
+            "regex": "^[0-9a-fA-F]+$",
```

Comment:
This is incorrect matching rule.

`objectInline.nesgtedArray` is not string field, but array, so regex is incorrect

Reply:
I think you have created a too complex test case for the test, which is too large and finally, there are difficulties to analyze a lot of snapshot lines.